recipes/double_conversion.rb

```diff
@@ -0,0 +1,49 @@
+#
+# Cookbook Name:: mcrouter
+# Recipe:: double_conversion
+#
+# Copyright 2015 EverTrue, Inc.
+# Copyright 2017 Roblox, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+dir = "#{Chef::Config[:file_cache_path]}/double-conversion"
+install_dir = '/usr/local'
+include_file = '/usr/local/lib/libdouble-conversion.a'
+
+git dir do
+  repository 'https://github.com/google/double-conversion.git'
+  revision node['mcrouter']['double_conversion_commit_hash']
+  action :sync
+  notifies :run, 'execute[cmake]'
+end
+
+execute 'cmake' do
```

Comment:
This comment applies to a lot of the `execute` blocks, but I would recommend giving them descriptive names (like `build and install double conversion`), not using the name of the command to execute. The reason why is that you have multiple resources in this cookbook that have the same name (like 'cmake') which really makes the resource collection confusing and probably prone to bugs.